src/Components/Table.css

```diff
@@ -0,0 +1,8 @@
+.table {
+   font-family: 'Quattrocento', serif;
```

Comment:
Just commenting here to give you another point of reference for my above comment about the reusable css classes. If you defined the class that I mentioned in my comment above, you could just add it to the table element directly, rather than having to write it out here again.

Reply:
This is really helpful, thanks!